src/App.css

```diff
@@ -0,0 +1,63 @@
+.nav__link {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__link-right {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link-links {
+  padding: 20px;
+}
+
+.border {
+  border-right: 2px solid #7d7a7a;
+}
+
+.title {
+  margin-left: 28%;
+  background: black;
+  color: violet;
+  width: 350px;
+}
+
+.container {
+  margin-left: 25%;
+  margin-right: 25%;
+  background: #e8e6e6;
+  border-collapse: collapse;
+}
+
+.container td {
+  font-weight: bold;
+}
+
+.inputs {
+  background: gray;
+  height: 60px;
+}
+
+.inputs td {
+  align-content: end;
+  color: rgb(72, 73, 178);
+  padding-left: 30%;
+}
+
+button {
+  font-size: 20px;
+  width: 100px;
+  height: 40px;
+  border: none;
+}
+
+.operation {
+  background: rgb(244, 160, 5);
+}
+
+.zero {
+  padding-left: 102px;
+  padding-right: 102px;
+}
```

Comment:
- [OPTIONAL] I highly encourage you to improve your website style by using attractive colors, and animation. This will help you to attract and impress anyone who is visiting your website to look more at you which may lead to employment. I highly recommend you come up with eye attractive design.